### PR TITLE
[Nim] Introduce httpbeast

### LINKED ISSUE #295: Add httpbeast
https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/Nim/httpbeast

### PR DESCRIPTION
**Pull request by waghanza**
Hi @dom96,

This `PR` adds https://github.com/dom96/httpbeast.

Could you review / approve this `PR` ?

Regards,

-----------

Closes #295

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -221,6 +221,11 @@ jobs:
     language: crystal
     env: FRAMEWORK=http4s
     services: docker
+  - stage: test
+    script: bash .ci/test.sh
+    language: crystal
```
Comment (dom96) [inline .travis.yml]: 👀

File: nim/httpbeast/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM nimlang/nim:0.20.2
+
+WORKDIR /usr/src/app
+
+COPY server.nim server.nimble ./
+
+RUN nimble c -d:release --threads:on -y server.nim
```
Comment (dom96) [inline nim/httpbeast/Dockerfile]: for extra speed nowadays, you can do ``-d:danger`` ;)

Comment (waghanza) [reply nim/httpbeast/Dockerfile]: if you are OK with that I'll mark this case as resolved without doing anythin
in the next `PR` I add the options @see https://github.com/the-benchmarker/web-frameworks/pull/1578/files#diff-80d3cc452e003fc3abf65e23df5f9256
the purpose is to compile `jester` and `httpbeast` the same way

Comment (dom96) [reply nim/httpbeast/Dockerfile]: do it whichever way you prefer, just note that `-d:danger` will give the best performance on latest Nim.

**Pull request merged**